demos/alphapose/writer.py

```diff
@@ -22,7 +22,7 @@
 EVAL_JOINTS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16]
 
 detection_threshold = 0.2
-keypoint_dist_threshold = None
+keypoint_dist_threshold = 10
```

Comment:
I made this change for clarity reasons. For me, it was hard to understand the `None` and how it would work when using it. I changed it to `10` and checked that the output didn't change.